### PR TITLE
Enable iic

### PR DESCRIPTION
**Pull request by podhrmic**

### CODE REVIEW

**Comment by podhrmic** [comment]
@heshamelmatary it still does not compile with IIC enabled - perhaps I am missing something in the waf script?

File: demo/cyberphys/main_fett.c [hunk 1]
```diff
@@ -274,16 +274,16 @@ void main_fett(void)
...
     startNetwork();
 
     funcReturn = xTaskCreate(prvMainTask, "prvMainTask", MAINTASK_STACK_SIZE, NULL, MAINTASK_PRIORITY, NULL);
 
     if (funcReturn == pdPASS) {
-        FreeRTOS_printf(("%s (Info)~  main_fett: Created prvMainTask successfully.<<<\r\n", getCurrTime()));
+        FreeRTOS_printf(("%s (Info)~  main_besspin: Created prvMainTask successfully.<<<\r\n", getCurrTime()));
     } else {
-        FreeRTOS_printf(("%s (Error)~  main_fett: Failed to create prvMainTask.<<<\r\n", getCurrTime()));
+        FreeRTOS_printf(("%s (Error)~  main_besspin: Failed to create prvMainTask.<<<\r\n", getCurrTime()));
```
Comment (jrtc27) [inline demo/cyberphys/main_fett.c:286]: These seem wrong when it's still main_fett.c and the function is called main_fett

Comment (podhrmic) [reply demo/cyberphys/main_fett.c:286]: I wanted to do just the minimal changes so the program can be booted in besspin-tool. The proper fix would be to either integrate the CHERI changes into our fork, or update the CHERI demos with the new cyberphys demo.

Comment (podhrmic) [reply demo/cyberphys/main_fett.c:286]: The main change being renaming `*fett` to `*besspin`

**Comment by heshamelmatary** [comment]
I have merged this PR with master, with a few additions.

**Pull request closed without merging**